dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/attachment/AttachmentUtil.java

```diff
@@ -96,13 +103,17 @@ public DataContentHandler createDataContentHandler(String mimeType,
                 DataSource ds) {
             DataContentHandler dch = super.createDataContentHandler(mimeType);
             if (dch == null) {
+	        if (LOG.isLoggable(Level.FINE)) { //Liberty Change Start
+	           LOG.fine("createDataContentHandler using DEFAULT_COMMAND_MAP for DataSource: " + ds.getName());
```

Comment:
Should ds be null checked to avoid an NPE here?

Reply:
I am not sure, but I will add a check for null to be safe.